docs/faq/faq.md

```diff
@@ -301,14 +301,17 @@ The Accelerator provides 3 mechanisms to enable utilizing certificates with ALB'
           "name": "My-Cert",
           "type": "import",
           "priv-key": "certs/example1-cert.key",
-          "cert": "certs/example1-cert.crt"
+          "cert": "certs/example1-cert.crt",
+          "chain": "certs/example1-cert.chain"
```

Comment:
Question - do you think this will be confusing for those that do not wish to use a chain?  Should we provide both examples, or do you think it's clear enough to remove the additional line when not required?